v139

lexlog opened the pull request.

maxvisser commented:
some clutter css can go now still; still need to remove this

jamalv commented:
Laptop screenshot is too far to the right:

<img width="1680" alt="Screenshot 2019-05-17 at 05 39 19" src="https://user-images.githubusercontent.com/30194799/57901649-51730300-7866-11e9-9680-8a687ad460ae.png">

maxvisser commented:
Is this also on develop?

maxvisser commented:
If so I will make a new branch for fixing all issues with MacBook.

jamalv commented:
Yes, also on develop

maxvisser commented:
done.

aghimpu committed:
add feature for excluding funds from pre-check results

aghimpu committed:
add fund formula products details to pre-check

dev-rminds committed:
merge develop into pre-check-exclude-funds-feature

dev-rminds committed:
Merge branch 'develop' into pre-check-add-formula-product-details

dev-rminds committed:
fund formula products details to pre-check refactoring

ateamcms committed:
add control type for record types

aghimpu committed:
pre-check fund exclude feature code refactoring

dev-rminds committed:
Merge branch 'develop' into pre-check-exclude-funds-feature

ateamcms committed:
fix exception with translator in notifications dropdown

dev-rminds committed:
merge develop into pre-check-exclude-funds-feature

dev-rminds committed:
pre-check fund exclusions refactoring and fixes

dev-rminds committed:
pre-check fund exclusions cleanup

dev-rminds committed:
Merge branch 'develop' into feature.add-control-type-for-record-types

irinaBerendeeva87 committed:
Merge branch 'develop' into pre-check-exclude-funds-feature

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 199.16kb -> 138.29kb (30.56%)

/react/assets/forus-webshop/resources/webshop-berg_en_dal/assets/img/placeholders/product-large.png -- 9.39kb -> 3.06kb (67.47%)
/react/assets/forus-webshop/resources/webshop-berg_en_dal/assets/img/placeholders/fund-large.png -- 14.76kb -> 5.33kb (63.91%)
/react/assets/forus-webshop/resources/webshop-berg_en_dal/assets/img/placeholders/organization-large.png -- 9.20kb -> 3.57kb (61.17%)
/react/assets/forus-webshop/resources/webshop-berg_en_dal/assets/img/placeholders/office-large.png -- 17.04kb -> 6.78kb (60.23%)
/react/assets/forus-webshop/resources/webshop-berg_en_dal/assets/img/placeholders/fund-thumbnail.png -- 3.33kb -> 1.66kb (50.16%)
/react/assets/forus-webshop/resources/webshop-berg_en_dal/assets/img/placeholders/office-thumbnail.png -- 3.38kb -> 1.88kb (44.44%)
/react/assets/forus-webshop/resources/webshop-berg_en_dal/assets/img/placeholders/product-thumbnail.png -- 1.59kb -> 0.91kb (42.66%)
/react/assets/forus-webshop/resources/webshop-berg_en_dal/assets/img/placeholders/organization-thumbnail.png -- 1.88kb -> 1.11kb (40.69%)
/react/assets/forus-webshop/resources/webshop-berg_en_dal/assets/img/sprite-map.png -- 136.06kb -> 111.53kb (18.03%)
/react/assets/forus-webshop/resources/_webshop-common/assets/img/file-icon-jpeg.svg -- 2.53kb -> 2.47kb (2.43%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

irinaBerendeeva87 committed:
Merge branch 'develop' into pre-check-exclude-funds-feature

MariaForus committed:
Text changes

MariaForus committed:
Small text update

dependabot[bot] committed:
Bump @typescript-eslint/eslint-plugin from 8.12.2 to 8.16.0

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 8.12.2 to 8.16.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v8.16.0/packages/eslint-plugin)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/eslint-plugin"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @react-google-maps/api from 2.19.3 to 2.20.3

Bumps [@react-google-maps/api](https://github.com/JustFly1984/react-google-maps-api/tree/HEAD/packages/react-google-maps-api) from 2.19.3 to 2.20.3.
- [Changelog](https://github.com/JustFly1984/react-google-maps-api/blob/develop/packages/react-google-maps-api/CHANGELOG.md)
- [Commits](https://github.com/JustFly1984/react-google-maps-api/commits/HEAD/packages/react-google-maps-api)

---
updated-dependencies:
- dependency-name: "@react-google-maps/api"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump react-router-dom from 6.27.0 to 7.0.1

Bumps [react-router-dom](https://github.com/remix-run/react-router/tree/HEAD/packages/react-router-dom) from 6.27.0 to 7.0.1.
- [Release notes](https://github.com/remix-run/react-router/releases)
- [Changelog](https://github.com/remix-run/react-router/blob/main/packages/react-router-dom/CHANGELOG.md)
- [Commits](https://github.com/remix-run/react-router/commits/react-router-dom@7.0.1/packages/react-router-dom)

---
updated-dependencies:
- dependency-name: react-router-dom
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @dnd-kit/core from 6.1.0 to 6.2.0

Bumps [@dnd-kit/core](https://github.com/clauderic/dnd-kit/tree/HEAD/packages/core) from 6.1.0 to 6.2.0.
- [Release notes](https://github.com/clauderic/dnd-kit/releases)
- [Changelog](https://github.com/clauderic/dnd-kit/blob/master/packages/core/CHANGELOG.md)
- [Commits](https://github.com/clauderic/dnd-kit/commits/@dnd-kit/core@6.2.0/packages/core)

---
updated-dependencies:
- dependency-name: "@dnd-kit/core"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump eslint-plugin-react from 7.36.1 to 7.37.2

Bumps [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) from 7.36.1 to 7.37.2.
- [Release notes](https://github.com/jsx-eslint/eslint-plugin-react/releases)
- [Changelog](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/CHANGELOG.md)
- [Commits](https://github.com/jsx-eslint/eslint-plugin-react/compare/v7.36.1...v7.37.2)

---
updated-dependencies:
- dependency-name: eslint-plugin-react
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump i18next from 23.15.1 to 24.0.2

Bumps [i18next](https://github.com/i18next/i18next) from 23.15.1 to 24.0.2.
- [Release notes](https://github.com/i18next/i18next/releases)
- [Changelog](https://github.com/i18next/i18next/blob/master/CHANGELOG.md)
- [Commits](https://github.com/i18next/i18next/compare/v23.15.1...v24.0.2)

---
updated-dependencies:
- dependency-name: i18next
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dev-rminds committed:
Merge branch 'develop' into pre-check-exclude-funds-feature

dev-rminds committed:
precheck fund exclusions feedback

dev-rminds committed:
update translations

irinaBerendeeva87 committed:
Merge branch 'develop' into pre-check-exclude-funds-feature

dev-rminds committed:
add external funds to pre-check editor and fix flickering icon when console is open

dev-rminds committed:
increase funds count on payouts page

dev-rminds committed:
update reimbursement edit and details title to h1

irinaBerendeeva87 committed:
Merge branch 'develop' into fixes.exception-with-translator-in-notifications-dropdown

dev-rminds committed:
epic profiles

dev-rminds committed:
Merge branch 'develop' into epic.profiles

dev-rminds committed:
profiles minor visual and consistency fixes

dev-rminds committed:
update sponsor auth redirect by permissions

dev-rminds committed:
fix compatibility with react-router-dom v7

dev-rminds committed:
fix profile records responsivity  on webshop

Commit:
Merge pull request #485 from teamforus/dependabot/npm_and_yarn/develop/i18next-24.0.2

Bump i18next from 23.15.1 to 24.0.2

Commit:
Merge pull request #484 from teamforus/dependabot/npm_and_yarn/develop/eslint-plugin-react-7.37.2

Bump eslint-plugin-react from 7.36.1 to 7.37.2

dependabot[bot] committed:
Bump i18n-iso-countries from 7.12.0 to 7.13.0

Bumps [i18n-iso-countries](https://github.com/michaelwittig/node-i18n-iso-countries) from 7.12.0 to 7.13.0.
- [Release notes](https://github.com/michaelwittig/node-i18n-iso-countries/releases)
- [Commits](https://github.com/michaelwittig/node-i18n-iso-countries/compare/v7.12.0...v7.13.0)

---
updated-dependencies:
- dependency-name: i18n-iso-countries
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump typescript from 5.5.4 to 5.7.2

Bumps [typescript](https://github.com/microsoft/TypeScript) from 5.5.4 to 5.7.2.
- [Release notes](https://github.com/microsoft/TypeScript/releases)
- [Changelog](https://github.com/microsoft/TypeScript/blob/main/azure-pipelines.release.yml)
- [Commits](https://github.com/microsoft/TypeScript/compare/v5.5.4...v5.7.2)

---
updated-dependencies:
- dependency-name: typescript
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Commit:
Merge pull request #482 from teamforus/dependabot/npm_and_yarn/develop/typescript-5.7.2

Bump typescript from 5.5.4 to 5.7.2

Commit:
Merge pull request #483 from teamforus/dependabot/npm_and_yarn/develop/i18n-iso-countries-7.13.0

Bump i18n-iso-countries from 7.12.0 to 7.13.0

Commit:
Merge pull request #477 from teamforus/dependabot/npm_and_yarn/develop/typescript-eslint/eslint-plugin-8.16.0

Bump @typescript-eslint/eslint-plugin from 8.12.2 to 8.16.0

Commit:
Merge pull request #478 from teamforus/dependabot/npm_and_yarn/develop/react-google-maps/api-2.20.3

Bump @react-google-maps/api from 2.19.3 to 2.20.3

Commit:
Merge pull request #479 from teamforus/dependabot/npm_and_yarn/develop/react-router-dom-7.0.1

Bump react-router-dom from 6.27.0 to 7.0.1

Commit:
Merge pull request #480 from teamforus/dependabot/npm_and_yarn/develop/dnd-kit/core-6.2.0

Bump @dnd-kit/core from 6.1.0 to 6.2.0

dependabot[bot] committed:
Bump @dnd-kit/sortable from 8.0.0 to 10.0.0

Bumps [@dnd-kit/sortable](https://github.com/clauderic/dnd-kit/tree/HEAD/packages/sortable) from 8.0.0 to 10.0.0.
- [Release notes](https://github.com/clauderic/dnd-kit/releases)
- [Changelog](https://github.com/clauderic/dnd-kit/blob/master/packages/sortable/CHANGELOG.md)
- [Commits](https://github.com/clauderic/dnd-kit/commits/@dnd-kit/sortable@10.0.0/packages/sortable)

---
updated-dependencies:
- dependency-name: "@dnd-kit/sortable"
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Commit:
Merge pull request #497 from teamforus/dependabot/npm_and_yarn/develop/dnd-kit/sortable-10.0.0

Bump @dnd-kit/sortable from 8.0.0 to 10.0.0

Commit:
Merge pull request #467 from teamforus/imgbot

[ImgBot] Optimize images

lexlog committed:
fix prettier

Commit:
Merge pull request #474 from teamforus/text-changes

Text changes

irinaBerendeeva87 committed:
Merge branch 'develop' into epic.profiles

Commit:
Merge pull request #489 from teamforus/update-wcag-headers-on-reimbursements

Update wcag headers on reimbursements

Commit:
Merge pull request #488 from teamforus/fixes.payouts-funds-list

increase funds count on payouts page

Commit:
Merge pull request #437 from teamforus/fixes.exception-with-translator-in-notifications-dropdown

Fix exception with translator in notifications dropdown

Commit:
Merge pull request #411 from teamforus/feature.add-control-type-for-record-types

Add control type for record types

Commit:
Merge pull request #389 from teamforus/pre-check-add-formula-product-details

add fund formula products details to pre-check

dev-rminds committed:
merge develop into pre-check-exclude-funds-feature and fix the conflicts

dev-rminds committed:
revert react-router-dom to 6.27.0

Commit:
Merge pull request #498 from teamforus/revert.react-router-dom

revert react-router-dom to 6.27.0

lexlog committed:
ignore minor versions dependabot

dev-rminds committed:
Merge pull request #499 from teamforus/feature.configure_dependabot

ignore minor versions dependabot

irinaBerendeeva87 committed:
Merge branch 'develop' into epic.profiles

Commit:
Merge pull request #383 from teamforus/pre-check-exclude-funds-feature

add feature for excluding funds from pre-check results

dev-rminds committed:
profiles feedback

dev-rminds committed:
merge develop into epic.profiles and fix conflicts

dev-rminds committed:
webshop fix products subcategory filter

dev-rminds committed:
update translations

dev-rminds committed:
npm audit fix

dev-rminds committed:
downgrading typescript to latest version supported by @typescript-eslint/eslint-plugin

dev-rminds committed:
remove no lonter needed typings and @babel/plugin-proposal-class-properties

Commit:
Merge pull request #503 from teamforus/fix.webshop-products-subcategory-filter

webshop fix products subcategory filter

Commit:
Merge pull request #504 from teamforus/npm-audit-fix

update and remove no longer needed dependencies

Commit:
Merge pull request #493 from teamforus/epic.profiles

Epic.profiles
